eng/common/scripts/Helpers/DocMs-Metadata-Helper.ps1

```diff
@@ -0,0 +1,55 @@
+function Generate-AadToken ($TenantId, $ClientId, $ClientSecret) {
```

Comment:
nit: Please put the `{` on its own line for functions (or any larger code blocks.